src/plugins/controls/common/control_group/control_group_diff_system.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import deepEqual from 'fast-deep-equal';
+import { omit } from 'lodash';
+import { OptionsListEmbeddableInput, OPTIONS_LIST_CONTROL } from '../options_list/types';
+import { RANGE_SLIDER_CONTROL } from '../range_slider/types';
+import { TIME_SLIDER_CONTROL } from '../time_slider/types';
+
+import { ControlGroupDiffSystem, ControlPanelState } from './types';
+
+interface DiffSystem {
+  getPanelIsEqual: (initialInput: ControlPanelState, newInput: ControlPanelState) => boolean;
+}
+
+const genericControlPanelDiffSystem: DiffSystem = {
+  getPanelIsEqual: (initialInput, newInput) => {
+    return deepEqual(initialInput, newInput);
+  },
+};
+
+export const ControlPanelDiffSystems: {
```

Comment:
This `ControlPanelDiffSystems` should be temporary and will be removed/cleaned up as part of the Portable Dashboards project.

Basically, if we had access to the `ControlGroupContainer` during the dashboard diff process rather than just `PersistableControlGroupInput`, we could then make use of the embeddable `getExplicitInputIsEqual` function for each child embeddable of the control group. However, this is a fairly large refactor that does not fit in to this PR, so the `ControlPanelDiffSystems` serves as a temporary workaround.